src/theme/variables.scss

```diff
@@ -84,5 +84,23 @@ $colors: (
 // Fonts
 // --------------------------------------------------
 
-@import "roboto";
-@import "noto-sans";
+// @import "roboto";
+// @import "noto-sans";
+
+@font-face {
+  font-family: 'Champagne-Bold';
+  font-style: normal;
+  font-weight: 900;
+  src: url('../assets/fonts/Champagne-Bold.ttf');
```

Comment:
you have two different tab spacings in this file (two spaced tabs and 4 space tabs). If you aren't already you should add the editconfig plugin to vscode and it should perform auto formatting for these kinds of things.